lib/assets/components/campaignTile.dart

```diff
@@ -81,43 +81,46 @@ class _CampaignTileState extends State<CampaignTile> {
         crossAxisAlignment: CrossAxisAlignment.start,
         children: <Widget>[
           // Image
-          Stack(children: <Widget>[
-            Container(
-              height: 120,
-              child: Padding(
-                padding: EdgeInsets.all(0),
-                child: Container(
-                    decoration: BoxDecoration(
-                  image: DecorationImage(
-                    image: customNetworkImageProvider(
-                        widget.campaign.getHeaderImage()),
-                    fit: BoxFit.cover,
-                  ),
-                  //borderRadius: BorderRadius.only(topLeft: Radius.circular(10))
-                )),
+          Expanded(
+            child: Stack(
+              children: <Widget>[
+              Container(
+                height: MediaQuery.of(context).size.height * 0.3,
```

Comment:
I haven't taken a proper look but I'm not sure if this will fix the issue? Have you been able to test on a smaller device?

Reply:
I've tried on smaller devices on Simulator and it works just fine.I also tried adding more words but it still is good to go.